### PR TITLE
Added Keyword Cipher doc

### LINKED ISSUE #110: Content: Key Word Cipher
## Description

Add the article for Key Word Cipher. Use the following structure: 
- Introduction
- Explanation
- Algorithm
- Code
- Sample I/O
- Complexity Analysis

### PR DESCRIPTION
**Pull request by nagasrihith608**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
yes
(Write your answer here.)

### Description
Added Keyword Cipher doc
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Fixed #110 
(Write your answer here.)

### CODE REVIEW

**Comment by nagasrihith608** [comment]
@HarshCasper please have a look into this pr. Thank You!!

File: docs/cryptography/keyword-cipher.mdx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+slug: /cryptography/keyword-cipher
+---
+
+## Introduction
+
+A keyword cipher is a form of monoalphabetic substitution. Here we use a keyword to determine the letter matchings of the cipher alphabet to the plain alphabet. Repetition of letters in the word is avoided and the cipher alphabet is generated with the keyword matching to remaining alphabets. 
+
+## Explanation
+
+We map each alphabet in the keyword to a corresponding alphabet from the alphabetical order set (A, B, C, etc ) avoiding any repetition. Now we map the remaining elements from the alphabetical set with the remaining alphabets.
```
Comment (HarshCasper) [inline docs/cryptography/keyword-cipher.mdx:12]: Keep `A`, `B`, `C` in backticks

**Review by HarshCasper — CHANGES_REQUESTED** [review]
The rest looks absolutely good to me. Would a follow-up to this issue would be to add the decryption part as well?

**Comment by nagasrihith608** [comment]
> The rest looks absolutely good to me. Would a follow-up to this issue would be to add the decryption part as well?

so like can we open another issue to improve this after this is merged?

**Comment by HarshCasper** [comment]
> > The rest looks absolutely good to me. Would a follow-up to this issue would be to add the decryption part as well?
> 
> so like can we open another issue to improve this after this is merged?

Yeah.

**Comment by nagasrihith608** [comment]
> > > The rest looks absolutely good to me. Would a follow-up to this issue would be to add the decryption part as well?
> > 
> > 
> > so like can we open another issue to improve this after this is merged?
> 
> Yeah.

ok sure after u merge this I will open new issue to decrypt

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**